counter-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: ticktock
```

Comment:
I really need to understand what this "default" value is for then.

Is it so that integration tests (which this module currently doesn't have) don't fail?
Or is it so that you can easily wire 2 modules together out of the box?
If the latter, using a module-derived name obviously won't work.